internal/magic/binary.go

```diff
@@ -155,3 +153,39 @@ func Marc(raw []byte, limit uint32) bool {
 // | g   l   T   F    | 1                | ...      |
 var Glb = prefix([]byte("\x67\x6C\x54\x46\x02\x00\x00\x00"),
 	[]byte("\x67\x6C\x54\x46\x01\x00\x00\x00"))
+
+// TzIf matches a Time Zone Information Format (TZif) file.
+// See more: https://tools.ietf.org/id/draft-murchison-tzdist-tzif-00.html#rfc.section.3
+// Its header structure is shown below:
+// +---------------+---+
+// |  magic    (4) | <-+-- version (1)
+// +---------------+---+---------------------------------------+
+// |           [unused - reserved for future use] (15)         |
+// +---------------+---------------+---------------+-----------+
+// |  isutccnt (4) |  isstdcnt (4) |  leapcnt  (4) |
+// +---------------+---------------+---------------+
+// |  timecnt  (4) |  typecnt  (4) |  charcnt  (4) |
+func TzIf(raw []byte, limit uint32) bool {
+	// File is at least 44 bytes (header size).
+	if len(raw) < 44 {
+		return false
+	}
+
+	if !bytes.HasPrefix(raw, []byte("TZif")) {
+		return false
+	}
+
+	// Field "typecnt" MUST not be zero.
+	if bytes.Equal(raw[36:40], []byte{0x00}) {
+		return false
+	}
```

Comment:
```suggestion
	if binary.LittleEndian.Uint32(raw[36:40]) == 0 {
		return false
	}
```
The specification says:
_NOTE: All multi-octet integer values MUST be stored in big-endian format (high-order octet first)._
So you have to interpret those 4 bytes as a big-endian value.